style.css

```diff
@@ -4,19 +4,126 @@
     padding: 0; 
     box-sizing: border-box;
 }
-html{
-    font-size: 12px; 
-}
-body{
-    display: flex; 
-    flex-direction: column;
+* {
+    padding: 0;
+    margin: 0;
+    font-family: sans-serif;
+  }
+  header {
+    display: flex;
+    position: fixed;
+    justify-content: space-around;
+    position: absolute;
+    width: 100%;
+    background: rgba(55, 64, 85, 0.5);
+    /*     background: transparent;
+     */
+    transition: all 0.5s;
+    z-index: 997;
+    padding: 20px 0px;
+  }
+  .align-class{
+      position: relative;
+  }
+  .baker-class {
+    display: flex;
+    font-size: 28px;
+    line-height: 1;
+    font-weight: 600;
+    letter-spacing: 2px;
+    font-family: "Poppins", sans-serif;
+    justify-content: space-around;
+    align-content: center;
+  }
+  nav ul li a,
+  header div a {
+    color: white;
+    cursor: pointer;
+    text-decoration: none;
+  }
+  li {
+    list-style: none;
+    padding: 10px 10px 10px 20px;
+  }
+  li a:hover {
+    color: #ffc107;
+  }
+  .ul-class {
+    display: flex;
+    justify-content: space-between;
+    text-align: center;
+    color: white;
+    font-size: 14px;
+    font-family: "Open Sans", sans-serif;
+  }
+  .fa-bars {
+    display: none;
+    margin-right: 10px;
+  }
+  li a:hover {
+    color: #ffc107;
+  }
+  .home-class {
+    width: 100%;
+    height: 80vh;
+    background-image: url(https://bootstrapmade.com/demo/themes/Baker/assets/img/hero-bg.jpg);
+    background-color: rgba(45, 53, 69, 0.7);
+    background-size: cover;
+    position: relative;
+    display: flex;
+    justify-content: center;
     align-items: center;
+  }
+  .home-class::before {
+    content: "";
+    background: rgba(45, 53, 69, 0.7);
+    position: absolute;
+    bottom: 0;
+    top: 0;
+    left: 0;
+    right: 0;
+  }
+  .content-class {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
     justify-content: center;
-    font-family: 'Open Sans', sans-serif;
-}
-h1,h2,h3,h4,h5,h6{
-    font-family: 'Raleway', sans-serif;
-}
-p{
-    font-family: 'Open Sans', sans-serif;
-}
\ No newline at end of file
+    align-items: center;
+    top: 50%;
+    left: 50%;
+    width: 100%;
+    padding: 50px 0px;
+    transform: translate(-50%, -50%);
+    letter-spacing: 1px;
+  }
+  .fomate-class {
+    text-align: center;
+    font-family: "Raleway", sans-serif;
+  }
+  div h1 {
```

Comment:
it would be better if you give the div another class name, and use it here instead of giving the style to the element itself.